queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name, neutered, escape_attempts FROM animals WHERE neutered = 'true' AND escape_attempts < 3;
+SELECT name, date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
```

Comment:
- [ ] in this case you should only return the date_of_birth

Reply:
Hi, @mikethreels
I've implemented the required changes.